css/style.css

```diff
@@ -0,0 +1,308 @@
+*{
+    box-sizing: border-box;
```

Comment:
The box-sizing property isn't supported in IE6 and IE7. Research and use vendor prefixes when declaring border-box.